app/helpers/session/check/session_check_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'devise'
+
+module Session
+  module Check
+    module SessionCheckHelper
+      def session_check(options = {})
+        locals = {
+          session_time: Devise.timeout_in,
+          check_every: 10,
+          reset_counter_on_ajax: true,
+          logged_out_url: '/users/sign_in',
+          current_user: current_user
```

Comment:
So this now also gets the current user as opposed to before. Just checking that was on purpose

Reply:
Yes, that's correct. Before the render happened "in the calling controller". But now with [Rails 5](https://evilmartians.com/chronicles/new-feature-in-rails-5-render-views-outside-of-actions) you can do ERB renders "outside" views and controllers without tons of shenanigans.

So, I now have to pass in "everything we need", which includes the `current_user`.